e2e_test/js-tests/test_viem_tx.mjs

```diff
@@ -178,7 +178,7 @@ describe("viem send tx", () => {
 			to: "0x00000000000000000000000000000000DeaDBeef",
 			value: 2,
 			feeCurrency: process.env.FEE_CURRENCY,
-			maxFeePerGas: 50000000000n,
+			maxFeePerGas: 50000000002n,
 			maxPriorityFeePerGas: 2n,
```

Comment:
Looking at the name of this test it seems like we should remove these gas price parameters